### PR TITLE
Add metadata functions for organizations and independent disks

### PR DESCRIPTION
**Pull request by adambarreiro**
## Description
This PR adds CRUD metadata methods for the AdminOrg, and Disk types.

### CODE REVIEW

File: govcd/api_vcd_test.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
-//go:build api || openapi || functional || catalog || vapp || gateway || network || org || query || extnetwork || task || vm || vdc || system || disk || lb || lbAppRule || lbAppProfile || lbServerPool || lbServiceMonitor || lbVirtualServer || user || search || nsxv || nsxt || auth || affinity || role || alb || certificate || vdcGroup || ALL
-// +build api openapi functional catalog vapp gateway network org query extnetwork task vm vdc system disk lb lbAppRule lbAppProfile lbServerPool lbServiceMonitor lbVirtualServer user search nsxv nsxt auth affinity role alb certificate vdcGroup ALL
+//go:build api || openapi || functional || catalog || vapp || gateway || network || org || query || extnetwork || task || vm || vdc || system || disk || lb || lbAppRule || lbAppProfile || lbServerPool || lbServiceMonitor || lbVirtualServer || user || search || nsxv || nsxt || auth || affinity || role || alb || metadata || certificate || vdcGroup || ALL
```
Comment (adambarreiro) [inline govcd/api_vcd_test.go:1]: This is done in #430

File: .changes/v2.15.0/438-features.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* Added support to set, get and delete metadata to AdminOrg with the methods
+  `AdminOrg.AddMetadataEntry`, `AdminOrg.AddMetadataEntryAsync`, `AdminOrg.GetMetadata`,
+  `AdminOrg.DeleteMetadataEntry` and `AdminOrg.DeleteMetadataEntryAsync`. [GH-438]
+* Added support to set, get and delete metadata to Disk with the methods
+  `Disk.AddMetadataEntry`, `Disk.AddMetadataEntryAsync`, `Disk.GetMetadata`,
+  `Disk.DeleteMetadataEntry` and `Disk.DeleteMetadataEntryAsync`. [GH-438]
```
Comment (lvirbalas) [inline .changes/v2.15.0/438-features.md:6]: Nit: not sure we had multiline entries in changelog before, and whether the link parsing ("[GH-438]") will work correctly with make-changelog.sh - maybe you could double check?

**Review by lvirbalas — APPROVED** [review]

**Review by Didainius — APPROVED** [review]
Looks great!

**Comment by adambarreiro** [comment]
Added also a function to get metadata from a regular Organization. Updated the tests accordingly.

**Review by mikeletux — APPROVED** [review]
LGTM! Approved :) Thanks @adambarreiro !

**Pull request merged**